packages/mcp-github-project-manager/github-project-manager.sh

```
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Get the directory of the script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# First try to load environment variables from dist/.env file if it exists
+if [ -f "$SCRIPT_DIR/dist/.env" ]; then
+    echo "Loading environment variables from $SCRIPT_DIR/dist/.env"
+    export $(grep -v '^#' "$SCRIPT_DIR/dist/.env" | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)